app.ts

```diff
@@ -0,0 +1,285 @@
+let fromParameter = 0;
+let timer: number;
+let recordCount: number;
+
+//// On Window Load
+window.onload = function () {
+    getRecordCount()
+        .then(() => {
+            return getHeadings();
+        })
+        .then(() => {
+            return getTable();
+        });
+    console.log("CALL HECTOR!!!");
+};
+
+function getParameters(fromParameter: number) {
+    let toParameter: number;
+    let noOfRows = getNoOfRows();
+
+    toParameter = fromParameter + noOfRows;
+    return toParameter;
+}
+
+function getNoOfRows(): number {
+    const height = window.innerHeight;
+    let noOfRows = Math.floor(height / 40);
+    return noOfRows;
+}
+
+//// Functions To Create/Clear The HTML
+// Heading Row
+function createHeadingRow(headingData: string) {
+    const heading: HTMLElement | null = document.getElementById("heading");
+
+    let headings = `<div class="headings" id="headings">${headingData}</div>`;
+    if (heading !== null) {
+        heading.innerHTML += headings;
+    }
+}
+
+// Table Content
+function createTableContent(contentData: string) {
```

Comment:
They way you are using `contentData` gives me the idea that it is not simply a `string` -_-